src/Ui/Menu/AdminMenuListener.php

```diff
@@ -19,7 +19,7 @@ public function __invoke(MenuBuilderEvent $event): void
         $salesMenu
             ->addChild('invoices', ['route' => 'sylius_invoicing_plugin_admin_invoice_index'])
                 ->setLabel('sylius_invoicing_plugin.ui.invoices')
-                ->setLabelAttribute('icon', 'inbox')
+				->setLabelAttribute('icon', 'file')
```

Comment:
Formatting issue. Also it would be nice to have a screenshot with proposed change :)

Reply:
Sure, I will do that on Thursday.